src/Explorer/Controls/TreeComponent/treeComponent.less

```diff
@@ -7,7 +7,7 @@
     }
 
     .treeNodeHeader {
-      margin: 1px;
+      margin: @SmallSpace 2px;
```

Comment:
I tested this and it still didn't look right. Please make sure you test these changes locally.

This is what it looks like with your change:
![image](https://user-images.githubusercontent.com/56978073/228676060-d2bd564a-3084-4631-8e82-34c8cd65cb3d.png)

This is what it looks like currently:
![image](https://user-images.githubusercontent.com/56978073/228676246-3456d5cb-1be7-4856-8de5-a4ce7a031695.png)

As you can see, the blue background color no longer covers the entire item due to the margin.

I looked into the issue myself and the root cause of the the outline not being entirely visible is that the parent div is exactly the same size as the `treeNodeHeader` div, and when the outlines are shown, the height and the width of the `treeNodeHeader` div becomes two pixels longer due to the width of the outline being 1px.

An easy fix is to add 1px of padding to the parent div which is `databaseHeader`. I tested this fix locally and it seemed to work:
![image](https://user-images.githubusercontent.com/56978073/228684662-cd9e142f-3af0-48bb-bea5-442ea5ceb7b0.png)

Reply:
![image](https://user-images.githubusercontent.com/56978073/228684853-94065190-eb6f-4cf8-855f-8a80ca742348.png)